src/System.Threading.Tasks.Dataflow/tests/project.json

```diff
@@ -27,7 +29,6 @@
     "netstandard1.3": {}
   },
   "supports": {
-    "coreFx.Test.netcore50": {},
```

Comment:
When Process.Start is available, should be easy. Until then, I'm not aware of any workaround.